src/logger/transports/bitdrift.ts

```diff
@@ -18,8 +18,7 @@ export const bitdriftTransport: Transport = (
 ) => {
   const log = logFunctions[level]
   log(message.toString(), {
-    // match Sentry payload
-    context,
+    __context__: context,
```

Comment:
Also noting I renamed this so that it's clearly "special" and doesn't conflict with some metadata we already have defined on some metric events.